infra/modules/service/database-access.tf

```diff
@@ -14,9 +14,24 @@ resource "aws_vpc_security_group_ingress_rule" "db_ingress_from_service" {
   referenced_security_group_id = aws_security_group.app.id
 }
 
+
 resource "aws_iam_role_policy_attachment" "app_db_access" {
   count = var.db_vars != null ? 1 : 0
 
   role       = aws_iam_role.service.name
   policy_arn = var.db_vars.access_policy_arn
 }
+
+resource "aws_iam_role_policy_attachment" "app_service_db_access" {
+  count = var.db_vars != null ? 1 : 0
+
+  role       = aws_iam_role.app_service.name
+  policy_arn = var.db_vars.app_access_policy_arn
+}
+
+resource "aws_iam_role_policy_attachment" "migrator_db_access" {
+  count = var.db_vars != null ? 1 : 0
+
+  role       = aws_iam_role.migrator_service.name
```

Comment:
I didn't catch this in the other PR, but we shouldn't name the role `migrator_service`. the migrations are run as a one-off task, not as a service. can you rename the role to something like `migrator` or `migrator_task`?

Reply:
Sounds good and done